Rakefile

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "rubocop/rake_task"
+require "puppet-lint/tasks/puppet-lint"
+require "r10k/action/puppetfile/install"
+require "rspec/core/rake_task"
+
+RuboCop::RakeTask.new
+PuppetLint::RakeTask.new :lint do |config|
+  config.ignore_paths = ["vendor/**/*", "modules/**/*"]
+  # Disabled checks should be removed after the puppet code has been updated.
+  config.disable_checks = %w[autoloader_layout
+                             documentation
+                             140chars
+                             double_quoted_strings
+                             arrow_alignment
+                             only_variable_string
+                             trailing_whitespace
+                             arrow_on_right_operand_line
+                             case_without_default]
+end
+
```

Comment:
GPLT includes a `pr` namespace with a `pr:check` task to allow running all of the pr checks with a single command. Should we do the same here?

Reply:
I'm a fan of this as well.  Last time we debated it we said we would go that pr:check route unless we had stuff that got significant time gains from being parallel.  So unless that is a factor I vote for the single task that does whatever should be done.

Reply:
:-1: I am not a fan of this.  I tried it with all of the checks in one travis run and it made it difficult to resolve issues in a timely manner.  Since the travis run will exit on the first failure, if the rubocop fails, then any additional failures in the puppet-lint and/or spec tests are not discovered until the full assembly of tests is made to pass in turn.  In this case, it tripled the amount of travis runs needed in order to discover the issues that were part of this PR.

Reply:
I don't mind if Travis runs them separately but it is helpful when testing locally to have a single task to run rather than having to consult the travis file to see what will be run and then running those separately myself.

Reply:
ok yeah, if it stops I guess we shouldn't do that in travis.  But having a single rake task for dev testing is still on my "would like to have" list. Wish we could somehow have both...

Reply:
@RandellP I have added a `test:all` task to do this.